packages/storybook/stories/extension/ShortcutLinks.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import ShortcutLinks from 'extension/src/newtab/ShortcutLinks';
+import ExtensionProviders from './_providers';
+import { Boot, BootCacheData } from '@dailydotdev/shared/src/lib/boot';
+import defaultUser from '@dailydotdev/shared/__tests__/fixture/loggedUser';
+import { Alerts } from '@dailydotdev/shared/src/graphql/alerts';
+import { RemoteSettings } from '@dailydotdev/shared/src/graphql/settings';
+import { ChecklistViewState } from '@dailydotdev/shared/src/lib/checklist';
+import { HttpResponse, http } from 'msw';
+
+const meta: Meta<typeof ShortcutLinks> = {
+  title: 'Extension/ShortcutLinks',
+  component: ShortcutLinks,
+  args: {
+    shouldUseListFeedLayout: true,
+  },
+  render: (args) => {
+    return (
+      <ExtensionProviders><ShortcutLinks {...args} /></ExtensionProviders>
+    );
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof ShortcutLinks>;
+
+export const ListFeedLayout: Story = {
```

Comment:
Not full against it, but somehow feel we should probably aim to make components agnostic to all our implementation details (eg they shouldn't really care about boot/settings/etc)

Would love to hear what @sshanzel / @nensidosari think about it?
(it's ok for the sake of this project perhaps to avoid scope creeping)

Reply:
Yes, I think we can actually refactor it to make the component as dumb as possible. We will make it such that the component would only accept an array of objects with link and icon properties. Then that is what we will only add in our Storybook. Completely detached from all the hooks/contexts.

Reply:
Couldn't agree more, that's why it is currently hard for us to actually just add all components we have in storybook, they are more complex then necessary.